lib/explorer/backend/lazy_series.ex

```diff
@@ -1053,6 +1054,13 @@ defmodule Explorer.Backend.LazySeries do
     Backend.Series.new(data, {:list, :string})
   end
 
+  @impl true
+  def split_into(series, by, fields) do
+    data = new(:split_into, [lazy_series!(series), by, fields], :string)
+
+    Backend.Series.new(data, {:list, :struct})
```

Comment:
I think this should be something like `{:struct, Enum.map(fields, &{&1, :string})}`